tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2266,6 +2284,12 @@ <h2>Show similarity to selected datapoint</h2>
           value: '',
           observer: 'labelFeatureSelected_',
         },
+        // Choose which model is used for counterfactual computation
+        nearestCounterfactualModel: {
+          type: Number,
+          value: 0,
```

Comment:
Is this number referring to an id of a model or an index? Perhaps a change to the prop name to better depict its nature might be nice.

Reply:
done!